examples/auth_example/auth_example_server/migrations/20240115074259905/migration.json

```diff
@@ -10,14 +10,14 @@
         "columns": [
           {
             "name": "id",
-            "columnType": 2,
```

Comment:
Technically this is a null-opt since module migrations are never used in the dependent projects. Only the project definition is used to create the migrations for the projects.

The change in the migration is mostly done for consistency if something is set up from a clean slate.